### PR TITLE
Add Notebooks and update Icons and Styles

### PR DESCRIPTION
**Pull request by nstogner**
* Change quickstart to use a small LLM - thinking we should use the smaller model b/c it facilitates the "quick" in the "quickstart" - thinking we should show off a larger LLM is a following guide.
* Reduced amount of info in the quickstart... thinking we should move most of the details to the longer guides.
* Generate documentation from Notebooks --> I am close to getting the quickstart noteboook testable... not quite there yet.
* Documentation is assuming that we will build and publish an installer container... Will submit an issue if we merge this PR.

### CODE REVIEW

File: docs/installation.ipynb [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7aa46d59-0439-47ff-a104-812607d0b71f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!git clone https://github.com/substratusai/substratus && cd ./substratus"
```
Comment (brandonjbjelland) [inline docs/installation.ipynb:54]: I've had soem trouble with in-line `cd`s in other notebooks. I think it's conventional to cd using [the magic command](https://ipython.readthedocs.io/en/stable/interactive/magics.html#magic-cd) on a dedicated instead:

```suggestion
    "!git clone https://github.com/substratusai/substratus",
    "%cd ./substratus"
```

File: docs/installation.ipynb [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+   "id": "fdde4876-4821-4ff8-a74e-6304a23af98f",
+   "metadata": {
+    "tags": [
+     "remove_cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.chdir('./substratus')"
```
Comment (brandonjbjelland) [inline docs/installation.ipynb:69]: Ahh and you ran into the issue here. Use the magic command above and this can go

File: docs/installation.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+---
+sidebar_position: 5
+---
+
+# Installation
+
+<!-- THE MARKDOWN (.md) FILE IS GENERATED FROM THE NOTEBOOK (.ipynb) FILE -->
```
Comment (brandonjbjelland) [inline docs/installation.md:7]: 🙌

File: docs/introduction.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 1
+slug: "/"
+---
+
+# Introduction
+
+Substratus is a cross cloud substrate for training and serving AI models. Substratus extends the Kubernetes control plane to orchestrate ML operations through the addition of new API endpoints: Model, ModelServer, Dataset, and Notebook.
```
Comment (brandonjbjelland) [inline docs/introduction.md:8]: `sed s/AI/ML/g`

```suggestion
Substratus is a cross cloud substrate for training and serving ML models. Substratus extends the Kubernetes control plane to orchestrate ML operations through the addition of new API endpoints: Model, ModelServer, Dataset, and Notebook.
```

File: docs/introduction.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+* Adopt best practice conventions by default.
+* Train pre-packaged state of the art models on your own datasets.
+* Leverage GitOps out of the box.
+
+## One Minute Intro Video
+
+<!-- TODO: 1 minute intro video -->
+
+<div class="video-container">
+  <iframe class="video" src="https://www.youtube.com/embed/dQw4w9WgXcQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (brandonjbjelland) [inline docs/introduction.md:24]: fuck you very much! 😆

File: docs/quickstart.ipynb [hunk 6]
```diff
@@ -0,0 +1,286 @@
...
+   "execution_count": null,
+   "id": "a0931d4f-8dd3-4fef-a9ca-1350691d6b4b",
+   "metadata": {
+    "tags": [
+     "remove_output"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "!gcloud version || open 'https://cloud.google.com/sdk/docs/install'"
```
Comment (brandonjbjelland) [inline docs/quickstart.ipynb:58]: Also checking for the gcloud auth plugin:

```suggestion
    "!gcloud version || open 'https://cloud.google.com/sdk/docs/install'",
    "!gke-gcloud-auth-plugin --version || gcloud components install gke-gcloud-auth-plugin"
```

File: docs/quickstart.ipynb [hunk 7]
```diff
@@ -0,0 +1,286 @@
...
+   "source": [
+    "The packaged model server ships with an API (for application integration) and a GUI interface (for debugging). You can now open up your browser at [http://localhost:8080](http://localhost:8080) and talk to your model!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "95a9966d-5679-4208-9b28-0323aa80cd79",
+   "metadata": {},
+   "source": [
+    "If you are interesting in continuing your journey through Substratus, take a look at the [Guided Walkthrough](./category/walkthrough)."
```
Comment (brandonjbjelland) [inline docs/quickstart.ipynb:212]:
```suggestion
    "If you are interested in continuing your journey through Substratus, take a look at the [Guided Walkthrough](./category/walkthrough)."
```

File: docusaurus.config.js [hunk 8]
```diff
@@ -7,11 +7,11 @@ const darkCodeTheme = require("prism-react-renderer/themes/dracula");
 /** @type {import('@docusaurus/types').Config} */
 const config = {
   title: "Substratus",
-  tagline: "Deploying and fine tuning LLMs made easy",
+  tagline: "Build, train, and serve LLMs and more!",
   favicon: "img/favicon.ico",
 
   // Set the production url of your site here
-  url: "https://substratus.ai",
+  url: "https://substratusai.github.io",
```
Comment (brandonjbjelland) [inline docusaurus.config.js:14]: Why not serve from substratus.ai?

**Review by brandonjbjelland — APPROVED** [review]
LGTM - minor fixes in comments

File: docusaurus.config.js (hunk 8 above)
Comment (nstogner) [reply docusaurus.config.js:14]: Worried about getting SEO dinged on the domain bc our links to substratus.git are broken

File: docs/installation.ipynb (hunk 1 above)
Comment (nstogner) [reply docs/installation.ipynb]: Thanks!

**Pull request merged**